drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -832,11 +832,11 @@ static int set_up_plls(void)
 
 int stm32_clock_control_init(const struct device *dev)
 {
-	int r = 0;
```

Comment:
r needs to be initialized in case CONFIG_CPU_CORTEX_M7 is not defined. This should fix CI failures

Reply:
@gautierg-st Thanks for your catch.
By the way, I decided to change the return path from `return r;` to `return 0;` instead.
To achieve unification among each change of `stm32_clock_control_init` on the other files.